common/src/main/scala/tsec/common/package.scala

```diff
@@ -17,13 +18,32 @@ package object common {
     override def getMessage: String = cause
   }
 
-
   trait StringNewt {
     type I <: String
 
     val is: Is[I, String]
   }
 
+  implicit final class TSecFloatOps(val v: Float) extends AnyVal {
+    def toBytes: Array[Byte] = ByteBuffer.allocate(4).putFloat(v).array()
```

Comment:
Is there a source for these numbers? Seem like magic. A comment at the top of the syntax block is fine

Reply:
:+1: